HoI4Mod/Rome_Strikes_Back/common/decisions/categories/00_decision_categories.txt

```diff
@@ -227,6 +227,13 @@ lar_local_recruitment = {
 	}
 }
 
+ENG_imperial_conference_decision = {

```

Comment:
Why are the imperial conference decisions being added (back) in?

Reply:
I know they were added to the converter recently, but if you had purposefully removed here I hadn't realized.

Reply:
No, I didn't remove them, I'm just curious why they'd be added when England doesn't have an empire.